.github/scripts/review_agent.py

```
@@ -117,6 +101,23 @@ def _call_review_llm(prompt: str, *, anthropic_key: str, nvidia_key: str) -> str
+            import anthropic as _anthropic
+            client = _anthropic.Anthropic(api_key=anthropic_key)
+            resp = client.messages.create(
+                model=OPUS_MODEL,
+                max_tokens=2048,
+                messages=[{"role": "user", "content": prompt}],
+            )
+            text = next((b.text for b in resp.content if b.type == "text"), "")
+            if text:
+                print(f"[review] Got response from {OPUS_MODEL} (Anthropic fallback)", flush=True)
+                return text
+        except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)